pages/index.js

```diff
@@ -9,10 +9,10 @@ export default () => (
   <MainLayout>
     <HeroUnit />
     <SectionSeparator first />
-    <section id='tickets'><TicketsSection /></section>
+    <section id='Tickets'><TicketsSection /></section>
```

Comment:
Please don’t change this file.

The smooth scrolling logic should reside in one file only and should not affect any other file.